docs/start.rst

```diff
@@ -30,7 +30,7 @@ The installation is quick and straightforward.
     export AIRFLOW_HOME=~/airflow
 
     # install from pypi using pip
-    pip install apache-airflow
+    sudo pip install apache-airflow
```

Comment:
I think it depends on your local setting. Usually when you do pip install the apache airflow command is installed to your local user directory (~/.local/bin usually) and you need to add it to the path if you have not done it already. 

I think if we change it here, We should rather explain that those are the options - not everyone has sudo access and they should still be able to run and install Airflow. 

Can you please double check @qzhang1 if that was the problem and possibly update the description?

Reply:
Ah, that might be the case. I created a fresh ubuntu 20.04 vm and `sudo apt-get install pip3` to machine. For my case, I found regular `pip install apache-airflow` to local/bin and calling `airflow initdb` will cause airflow command to not be recognized/ added to path. 

I don't know how to install pip to local user but I can look into it and see if that previous issue is reproducible.